code/game/machinery/ai_slipper.dm

```diff
@@ -24,9 +24,9 @@
 
 /obj/machinery/ai_slipper/update_icon()
 	if(stat & NOPOWER || stat & BROKEN)
-		icon_state = "motion0"
+		icon_state = "liquid_dispenser"
 	else
-		icon_state = disabled ? "motion0" : "motion3"
+		icon_state = disabled ? "liquid_dispenser" : "liquid_dispenser_on"
```

Comment:
Gonna be straight up. I don't know what this does. If this would lead to unintended behavior, don't merge yet.

Reply:
The `?` operator is essentially an inline if statement, where the thing being evaluated is to the left of the ?, and what is returned from it is to the right of the ?, with the first thing being what happens if it's evaluated to true, and the second one is for when it is false.

In the example above, it'll make `icon_state` equal to `"liquid_dispenser"` if `disabled` is true, otherwise `icon_state` will be `"liquid_dispenser_on"`.

Reply:
Thank you! That seems it works fine, then.